package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nopp",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "NoPP (No Prototype Pollution) – tiny helper to protect against Prototype Pollution vulnerabilities in your application regardless if they introduced in your own code or in 3rd-party code.",
```

Comment:
An idea to shorten the one line description on the package page here, felt a bit too long.

```suggestion
  "description": "NoPP (No Prototype Pollution) – tiny helper to protect against Prototype Pollution vulnerabilities in your application",
```